packages/themes/_playerUi.scss

```diff
@@ -625,7 +625,7 @@
     color: $primary-triad-color;
     font-family: "Open Sans", sans-serif;
     font-weight: bold;
-    max-height: 80vh;
+    max-height: calc(100vh - 300px);
```

Comment:
thanks @sanjayradadiya  - but what does the 300px represent?  Is it the height of a particular element?

Reply:
Not particular but mostly its total distance from document top to collapsible panel top

Reply:
![image](https://user-images.githubusercontent.com/16476120/123259148-c7e89180-d511-11eb-9c44-59dcbe9d7bd2.png)

Reply:
The fix works, but I like to try to avoid "magic numbers".  300px isn't the height of the red arrow in your diagram.

Could you research a little more why the value of 300 works please?

Reply:
yes, sure

Reply:
![Screenshot from June 25, 2021 3_45 PM](https://user-images.githubusercontent.com/16476120/123409947-633e3d00-d5cc-11eb-9589-a72b65a633b4.png)

Reply:
that's perfect, thanks.